### DISCUSSION TITLE
Does it support Archer T4U V5?

### DISCUSSION THREAD

**Discussion by justBA** [chris1111/Wireless-USB-Big-Sur-Adapter#265] [Q&A]
I just try to use Archer T4U V5 to connect wifi. But it seems not working ...

**Comment by chris1111** [comment]
There is no V5 for Archer T4U 
Only V 1,2,3

**Reply by EdwardLatour** [reply]
I just bought today a Archer T4U V5 Version - and it works fine!

V5 is documented on the website
https://www.tp-link.com/en/support/download/archer-t4u/

**Comment by chris1111** [comment]
V5 its only woking in windows

![Screenshot 2023-05-12 at 20 20 38](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/assets/6248794/848a5b3d-2a86-4c4c-b7f6-903b73e3c08b)
